.github/workflows/pr_master_test.yml

```diff
@@ -0,0 +1,397 @@
+name: GitHub CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  PYTHONWARNINGS: ignore::UserWarning
+  PYTHON_BASE_PKGS: >
```

Comment:
A thought - we should add `wheel` to the base packages. I noticed yesterday that we get a lot of these messages: 
```
Could not build wheels for pyodbc, since package 'wheel' is not installed.
Could not build wheels for pytz, since package 'wheel' is not installed.
```
It doesn't stop the package from building. That being said, I added `wheel` to my local branch, and it makes the setup somewhat faster. It's not a huge speed-up in test time, but I did notice a difference in when it's able to build and install from `wheel` as opposed to from source, and we might as well be efficient, right?

Reply:
Seems reasonable.  Done.